packages/@react-spectrum/s2/src/TagGroup.tsx

```diff
@@ -323,7 +323,45 @@ function TagGroupInner<T>({
                       style={item.props.UNSAFE_style}
                       key={item.key}
                       className={item.props.className({size, allowsRemoving: Boolean(onRemove)})}>
-                      {item.props.children({size, allowsRemoving: Boolean(onRemove), isInCtx: true})}
+                      <div
```

Comment:
since this is now essentially the same as TagWrapper, could we reuse it? Just don't want two copies of the same styling/structure that need to remain in sync